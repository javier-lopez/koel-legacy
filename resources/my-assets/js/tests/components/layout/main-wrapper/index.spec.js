import Component from '@/components/layout/main-wrapper/index.vue'

describe('component/layout/main-wrapper/index', () => {
  it('renders properly', async done => {
    const wrapper = await shallow(Component)
    wrapper.vm.$nextTick(() => {
      expect(wrapper).toMatchSnapshot()
      done()
    })
  })
})
