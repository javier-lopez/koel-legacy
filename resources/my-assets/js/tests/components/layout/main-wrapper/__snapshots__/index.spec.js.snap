// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component/layout/main-wrapper/index renders properly 1`] = `
<div id="mainWrapper">
  <sidebar-stub></sidebar-stub>
  <section id="mainContent">
    <!---->
    <div class="translucent" style="background-image: none;"></div>
    <!---->
    <!---->
    <!---->
    <!---->
    <!---->
    <!---->
    <!---->
    <!---->
    <!---->
    <!---->
    <!---->
    <!---->
    <!---->
    <!---->
  </section>
  <section id="extra" class="showing">
    <div class="tabs">
      <div class="header clear"><a class="lyrics active">Lyrics</a> <a class="artist">Artist</a> <a class="album">Album</a>
        <!---->
      </div>
      <div class="panes">
        <!---->
        <!---->
        <!---->
        <!---->
      </div>
    </div>
  </section>
  <div class="modal-wrapper">
    <!---->
    <!---->
    <!---->
    <!---->
  </div>
</div>
`;
